package.json

```diff
@@ -16,7 +16,7 @@
     "node": ">= 0.8.x"
   },
   "scripts": {
-    "download": "curl \"http://unicode.org/Public/9.0.0/ucd/UnicodeData.txt\" > UnicodeData.txt",
+    "download": "curl \"https://unicode.org/Public/9.0.0/ucd/UnicodeData.txt\" > UnicodeData.txt",
```

Comment:
me too. I already wanted to approve the pr ;) thx for the contribution anyways, though!